pipeline/transformer/transformer_calm/src/test/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformerTest.scala

```diff
@@ -254,6 +308,32 @@ class CalmTransformerTest extends FunSpec with Matchers {
     CalmTransformer(record, version) shouldBe a[Left[_, _]]
   }
 
+  it("surpresses data when Transmission=No") {
+    val a = calmRecord(
+      "Title" -> "abc",
+      "Level" -> "Collection",
+      "RefNo" -> "a/b/c",
+      "AltRefNo" -> "a.b.c",
+      "Transmission" -> "No"
+    )
+    val b = calmRecord(
+      "Title" -> "abc",
+      "Level" -> "Collection",
+      "RefNo" -> "a/b/c",
+      "AltRefNo" -> "a.b.c",
+      "Transmission" -> "Yes"
+    )
+    val c = calmRecord(
+      "Title" -> "abc",
+      "Level" -> "Collection",
+      "RefNo" -> "a/b/c",
+      "AltRefNo" -> "a.b.c",
+    )
+    CalmTransformer.shouldTransform(a) shouldBe false
+    CalmTransformer.shouldTransform(b) shouldBe true
+    CalmTransformer.shouldTransform(c) shouldBe true
```

Comment:
One more test case needed here: what if the Transmission field contains a completely unexpected value?

`"Transmission" -> "Manual"`, I dunno